android/app/src/main/java/com/expensify/chat/customairshipextender/CustomNotificationProvider.java

```diff
@@ -46,6 +46,8 @@
 
 public class CustomNotificationProvider extends ReactNotificationProvider {
 
+    private final Context context;
+
```

Comment:
Holding context references is dangerous in Android and a common cause of memory leaks. 

I can't see it being used outside of this provider, so let's just not hold a reference to it. Even if it was, that is also not a good idea. There are ways to get it safely, but it doesn't look like we need to do this anyway 👍

Reply:
I honestly don't remember why I added this